### PR TITLE
builders for hotfixes

### PR DESCRIPTION
**Pull request by dnfield**
This should automatically build new commits to hotfix branches and fix https://github.com/flutter/flutter/issues/49381

### CODE REVIEW

File: config/main.star [hunk 1]
```diff
@@ -138,7 +154,9 @@ def console_view(name, repo, refs = ['refs/heads/master'], exclude_ref = None):
 
 console_view('cocoon', COCOON_GIT)
 console_view('framework', FLUTTER_GIT)
+console_view('hotfix-framework', FLUTTER_GIT, [HOTFIX_REFS])
 console_view('engine', ENGINE_GIT)
```
Comment (godofredoc) [inline config/main.star]: Wouldn't this also include the hotfix_refs should we explicitly filter it?

File: config/main.star [hunk 2]
```diff
@@ -99,12 +100,27 @@ luci.gitiles_poller(
...
+  name = 'hotfix-gitiles-trigger-framework',
+  bucket = 'prod',
+  repo = FLUTTER_GIT,
+  refs = [HOTFIX_REFS],
+)
+
 luci.gitiles_poller(
   name = 'master-gitiles-trigger-engine',
   bucket = 'prod',
   repo = ENGINE_GIT,
```
Comment (godofredoc) [inline config/main.star]: should this also have refs to filter out hotfixes-ref and do not trigger tasks twice?

File: config/main.star (hunk 1 above)
Comment (dnfield) [reply config/main.star]: I don't think it does today but I don't mind being more explicit.

Comment (dnfield) [reply config/main.star]: Actually - it defaults to master: https://chromium.googlesource.com/infra/luci/luci-go/+/refs/heads/master/lucicfg/doc/README.md#luci.gitiles_poller

File: config/main.star (hunk 2 above)
Comment (dnfield) [reply config/main.star]: Ditto - not specifying it makes it default ot master.

**Pull request merged**